types/d3-scale-chromatic/index.d.ts

```diff
@@ -45,173 +48,269 @@ export const schemeSet3: string[];
 // Diverging
 // -----------------------------------------------------------------------
 /**
+ * Diverging color schemes are available as continuous interpolators (often used with d3.scaleSequential) and as discrete schemes
+ *  (often used with d3.scaleOrdinal). Each discrete scheme, such as d3.schemeBrBG, is represented as an array of arrays of hexadecimal
+ *  color strings. The kth element of this array contains the color scheme of size k; for example, d3.schemeBrBG[9] contains an array
+ *  of nine strings representing the nine colors of the brown-blue-green diverging color scheme. Diverging color schemes support
+ *  a size k ranging from 3 to 11.
+ *
  * Given a number value in the range [0,1], returns the corresponding color from the “BrBG” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateBrBG(value: number): string;
+
+export const schemeBrBG: string[][];
```

Comment:
We've had some issues with arrays (see, for example, the discussion in #18580). I think these should be `ReadonlyArray<ReadonlyArray<string>>` instead of `string[][]` to both prevent modification and relax the usage requirements.